newsfragments/1646.bugfix.rst

```diff
@@ -0,0 +1 @@
+fix ContractEvent type
```

Comment:
Maybe a *little* more than this :). Something like:

> `my_contract.events.MyEvent` was incorrectly annotated so that `MyEvent` was marked as a `ContractEvent` instance. Fixed to be a class type, ie~ `Type[ContractEvent]`.

Ideally, someone writing code that depends on web3 should be able to review the release notes, and roughly know what changes they need to make to their code. (even before clicking into the PR)

If you want to be fancy, you might use sphinx domains to link to the docs where contract events are discussed. I tend to put that effort into slightly more involved PRs. See an [example](https://py-evm.readthedocs.io/en/latest/release_notes.html#py-evm-0-3-0-alpha-15-2020-04-14) that is linked using [`:meth:` syntax](https://github.com/ethereum/py-evm/blame/6fef9c25eff570471d147a4f4a6d4eba00262f57/docs/release_notes.rst#L12) or a simple internal link. I tend to visit this reference pretty often: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html